release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -110,6 +110,10 @@ After you have deployed your cluster to run nodes with static IP addresses, you
 
 For more information, see the "Static IP addresses for vSphere nodes" section in the xref:../installing/installing_vsphere/installing-vsphere-installer-provisioned.adoc[Installing a cluster on vSphere] document.
 
+[id="ocp-4-14-bmo-validations"]
+==== Additional validation for the Bare Metal Host CR
+
+The Bare Metal Host Custom Resource (CR) now contains the `ValidatingWebhooks` parameter. The Bare Metal Operator will catch any configuration errors before accepting the CR, and will return a message with the configuration errors to the user.
```

Comment:
Might  want to make it clear that the webhook is what allows the operator to catch the errors (assuming I am right about that).
```suggestion
The Bare Metal Host Custom Resource (CR) now contains the `ValidatingWebhooks` parameter. With this parameter, the Bare Metal Operator now catches any configuration errors before accepting the CR, and returns a message with the configuration errors to the user.
```